products/kotlin.md

```diff
@@ -20,34 +20,39 @@ releaseDateColumn: true
 releaseColumn: true
 
 releases:
-  - releaseCycle: "1.6"
+-   releaseCycle: "1.6"
     cycleShortHand: 106
     eol: false #not sure about eol dates of kotlin if you find any information about this please change this part
     support: false
-    release: 2021-11-16
+    release: 2021-11-11
```

Comment:
Bumped an issue regarding this: https://youtrack.jetbrains.com/issue/TW-10730/Support-for-annotated-tags-for-Git

Since Kotlin doesn't use annotated tags, we get the date from the commit, which is a few days behind the official release date.